android/app/src/main/AndroidManifest.xml

```diff
@@ -1,8 +1,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
On the Keyman for Android app, we do this
https://github.com/keymanapp/keyman/blob/9c9696c5fdcf2e90c7c87ffa3bfc744263bf82c2/android/KMAPro/kMAPro/src/main/AndroidManifest.xml#L5-L6